core/ir/aarch64/instr_create_api.h

```diff
@@ -111,9 +111,20 @@
 #define OPND_CREATE_ZR(reg) \
     opnd_create_reg(opnd_get_size(reg) == OPSZ_4 ? DR_REG_WZR : DR_REG_XZR)
 
-/** Create an operand specifying LSL, the default shift type when there is no shift. */
+/** Create an operand specifying LSL (Logical Shift Left), the default shift type when
```

Comment:
style: `/**` on its own line (unless whole comment is one line)

Reply:
Done.